connectivity/netsocket/tests/TESTS/network/wifi/wifi_connect.cpp

```diff
@@ -30,6 +30,10 @@ using namespace utest::v1;
 void wifi_connect(void)
 {
     WiFiInterface *wifi = get_interface();
+    TEST_ASSERT(wifi);
+    if (wifi == NULL) {
```

Comment:
what's the difference between wifi tests returning (assert before) and asserting with a message (`TEST_ASSERT_NOT_NULL_MESSAGE` not in this case)

Reply:
Other tests are stopping as soon as a test assert comes.
This one doesn't stop, so it needs a "clean" return.